Squeezer/src/main/res/layout/slim_browser_layout.xml

```diff
@@ -23,53 +23,63 @@
     android:layout_height="fill_parent"
     android:orientation="vertical">
 
-    <LinearLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/parent_container"
-        android:layout_width="fill_parent"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:paddingBottom="8dp"
-        android:paddingStart="16dp"
-        android:paddingEnd="32dp"
-        android:paddingTop="8dp"
+        android:layout_marginBottom="8dp"
+        android:minHeight="?listPreferredItemHeight"
         android:visibility="gone"
         tools:visibility="visible">
 
         <ImageView
             android:id="@+id/icon"
             android:layout_width="@dimen/album_art_large_icon_size"
             android:layout_height="@dimen/album_art_large_icon_size"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintEnd_toStartOf="@id/text1"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toBottomOf="parent"
+            android:padding="4dp"
             android:contentDescription="@string/app_name"
-            android:layout_marginEnd="8dp"
             tools:src="@drawable/pending"/>
 
-        <LinearLayout
+        <TextView
+            android:id="@+id/text1"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
-            android:layout_weight="1"
-            android:gravity="center_vertical"
-            android:minHeight="48dp"
-            android:orientation="vertical"
-            android:paddingEnd="8dp">
-
-            <TextView
-                android:id="@+id/text1"
-                android:layout_width="fill_parent"
-                android:layout_height="wrap_content"
-                android:textAppearance="@style/SqueezerTextAppearance.ListItem.Primary"
-                tools:text="text1"/>
+            android:layout_marginStart="8dp"
+            app:layout_constraintStart_toEndOf="@id/icon"
+            app:layout_constraintEnd_toStartOf="@id/context_menu"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintBottom_toTopOf="@id/text2"
+            app:layout_constraintVertical_chainStyle="packed"
+            android:textAppearance="?textAppearanceHeadline6"
```

Comment:
Use of ?textAppearanceHeadline6 seems odd and inconsistent (bypasses styles.xml and uses
material styles, as opposed to the @android:style/TextAppearance the rest of the app uses).